src/app/components/common/Button/CopyButton/CopyButton.tsx

```diff
@@ -0,0 +1,19 @@
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+
+type CopyButtonProps = {
+    textCopied: boolean;
+    classes: any;
+    onClickHandler: any;
+};
```

Comment:
Need empty line between type and component definition

Reply:
done